src/main/resources/org/broadinstitute/barclay/helpTemplates/generic.ftl

```diff
@@ -4,7 +4,7 @@
 
 <div class='row-fluid' id="top">
 
-	<#include "common.html"/>
+	<#include "common.ftl"/>
```

Comment:
I think common.html should retain it's current name since its just html and isn't really a template. It's name will never be transformed by the doc process, so the end result would be a .html that includes a .ftl if we do this.